assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -96,33 +125,94 @@ const Cart = ( { cartItems = [], cartTotals = {} } ) => {
 				value: totalTax,
 			} );
 		}
-		const totalShipping = parseInt( cartTotals.total_shipping, 10 );
-		const totalShippingTax = parseInt( cartTotals.total_shipping_tax, 10 );
-		totalRowsConfig.push( {
-			label: __( 'Shipping:', 'woo-gutenberg-products-block' ),
-			value: DISPLAY_PRICES_INCLUDING_TAXES
-				? totalShipping + totalShippingTax
-				: totalShipping,
-			description: (
-				<Fragment>
-					{ __(
-						'Shipping to location',
-						'woo-gutenberg-products-block'
-					) + ' ' }
-					<ShippingCalculator
-						address={ shippingCalculatorAddress }
-						setAddress={ setShippingCalculatorAddress }
-					/>
-				</Fragment>
-			),
-		} );
-
+		if ( SHIPPING_ENABLED && isShippingCalculatorEnabled ) {
+			const totalShipping = parseInt( cartTotals.total_shipping, 10 );
+			const totalShippingTax = parseInt(
+				cartTotals.total_shipping_tax,
+				10
+			);
+			totalRowsConfig.push( {
+				label: __( 'Shipping:', 'woo-gutenberg-products-block' ),
+				value: DISPLAY_PRICES_INCLUDING_TAXES
```

Comment:
This value should be hidden if totals are hidden and there is no address yet, but can be a follow up if you want to log it.